modern/clang/Dockerfile

```
@@ -162,3 +162,6 @@ RUN sudo mv /tmp/gcc/lib64 /usr/local/ \
     && sudo rm /etc/ld.so.cache \
     && sudo ldconfig -C /etc/ld.so.cache \
     && conan profile new --detect --force default
+
+RUN conan install -r conancenter -g deploy -if /opt/conan -l /opt/conan/conan.lock binutils/2.37@ --build=missing \
+    && rm -rf /home/conan/.conan/data
```

```suggestion
RUN conan install -r conancenter -g deploy -if /opt/conan -l /opt/conan/conan.lock binutils/2.37@ \
    && rm -rf /home/conan/.conan/data
```

If we use a lockfile it is because the package is available in ConanCenter. Building things from sources kinda invalidates the idea.

---

I agree, but true is that we don't have that specific package configuration in ConanCenter. When I run that command, conan says there is no such package:

```
Installing package: binutils/2.37
Requirements
    binutils/2.37 from 'conancenter' - Downloaded
    readline/8.0 from 'conancenter' - Downloaded
    termcap/1.3.1 from 'conancenter' - Downloaded
    zlib/1.2.11 from 'conancenter' - Downloaded
Packages
    binutils/2.37:8e7714e05da644d1474247995163817b009f35ad - Download
    readline/8.0:0dd0336dbd4a73b0c2d71bd9b9bd75a1d84cd661 - Missing
    termcap/1.3.1:dfbe50feef7f3c6223a476cd5aeadb687084a646 - Missing
    zlib/1.2.11:dfbe50feef7f3c6223a476cd5aeadb687084a646 - Download

Installing (downloading, building) binaries...
ERROR: Missing binary: readline/8.0:0dd0336dbd4a73b0c2d71bd9b9bd75a1d84cd661
ERROR: Missing binary: termcap/1.3.1:dfbe50feef7f3c6223a476cd5aeadb687084a646

termcap/1.3.1: WARN: Can't find a 'termcap/1.3.1' package for the specified settings, options and dependencies:
- Settings: arch=x86_64, build_type=Release, compiler=gcc, compiler.version=11, os=Linux
- Options: fPIC=True, shared=False
- Dependencies: 
- Requirements: 
- Package ID: dfbe50feef7f3c6223a476cd5aeadb687084a646
```

Question is, will Tapaholes fit it for us at some point, or do we need to open a PR for both packages to trigger a new build?